tests/test_cmwavex.py

```diff
@@ -0,0 +1,177 @@
+from io import StringIO
+
+import numpy as np
+
+from pint.models import get_model
+from pint.fitter import Fitter
+from pint.simulation import make_fake_toas_uniform
+from pint.utils import cmwavex_setup
+from pint.models.chromatic_model import cmu
+
+import pytest
+import astropy.units as u
+
+
+def test_cmwavex():
+    par = """`
+        PSR              B1937+21
+        LAMBDA   301.9732445337270
+        BETA      42.2967523367957
+        PMLAMBDA           -0.0175
+        PMBETA             -0.3971
+        PX                  0.1515
+        POSEPOCH        55321.0000
+        F0    641.9282333345536244  1  0.0000000000000132
+        F1     -4.330899370129D-14  1  2.149749089617D-22
+        PEPOCH        55321.000000
+        DM               71.016633
+        CM                     0.1
+        TNCHROMIDX               4
+        UNITS                  TDB
+    """
+    m = get_model(StringIO(par))
+
+    with pytest.raises(ValueError):
```

Comment:
should this be a different test than the other pieces?

Reply:
split into a different test.